packages/kbn-ui-shared-deps-src/BUILD.bazel

```diff
@@ -16,6 +16,7 @@ webpack_cli(
     "@npm//babel-loader",
     "@npm//css-loader",
     "@npm//url-loader",
+    "@npm//@elastic/ecs",
```

Comment:
We have been having a discussion wether or not to include this in the shared deps async. Just wanted to point out if we want to have this in the shared deps in the end, the correct place to add it is in the other package for external npm modules called `@kbn/ui-shared-deps-npm` :)